osquery/sql/sql.cpp

```diff
@@ -127,16 +117,25 @@ QueryData SQL::selectFrom(const std::initializer_list<std::string>& columns,
                           ConstraintOperator op,
                           const std::string& expr) {
   PluginRequest request = {{"action", "generate"}};
-  {
-    // Create a fake content, there will be no caching.
-    QueryContext ctx;
-    ctx.constraints[column].add(Constraint(op, expr));
-    ctx.colsUsed = UsedColumns(columns);
-    TablePlugin::setRequestFromContext(ctx, request);
+  // Create a fake content, there will be no caching.
+  QueryContext ctx;
+  ctx.constraints[column].add(Constraint(op, expr));
+  if (columns.size() > 0) {
+    auto colsUsed = UsedColumns(columns);
+    colsUsed.insert(column);
+    ctx.colsUsed = colsUsed;
```

Comment:
Will it save time?
ctx.colsUsed = UsedColumns(columns);
ctx.colsUsed.insert(column);

Reply:
who knows?)